plone/schemaeditor/tests/browser_testing.zcml

```diff
@@ -1,6 +1,12 @@
 <configure
     xmlns="http://namespaces.zope.org/zope"
-    xmlns:browser="http://namespaces.zope.org/browser">
+    xmlns:browser="http://namespaces.zope.org/browser"
+    xmlns:zcml="http://namespaces.zope.org/zcml">
+
+    <!-- only load PageTemplates zcml in Zope 4 -->
+    <configure zcml:condition="installed Products.PageTemplates.engine">
```

Comment:
although tests did pass, shouldn't that be ``Products.PageTemplates`` rather than ``Products.PageTemplates.engine``?